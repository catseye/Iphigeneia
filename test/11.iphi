(* Test 'valueof ... in ...' *)

var a in var b in begin
    a := 10;
    b := valueof c in begin
        c := a * 2
    end + 7
end
