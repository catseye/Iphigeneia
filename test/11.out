b=27
a=10
